tips/TIP-0026/indexer-rest-api.yaml

```diff
@@ -0,0 +1,806 @@
+openapi: 3.0.3
+info:
+  title: IOTA UTXO Indexer REST API
+  description: This document specifies the REST API for IOTA UTXO indexers.
+  contact:
+    email: contact@iota.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+  version: 1.0.0
+externalDocs:
+  description: Find out more about IOTA
+  url: 'https://iota.org'
+servers:
+  - url: 'http://127.0.0.1:14265'
+tags:
+  - name: basic outputs
+    description: Query Basic Outputs.
+  - name: alias outputs
+    description: Query Alias Outputs.
+  - name: foundry outputs
+    description: Query Foundry Outputs.
+  - name: nft outputs
+    description: Query NFT Outputs.
+paths:
+  /api/plugins/indexer/v1/outputs/basic:
+    get:
+      tags:
+        - basic outputs
+      summary: Returns basic outputs filtered based on parameters.
+      description: Returns basic outputs filtered based on parameters.
+      parameters:
+        - in: query
+          name: address
+          schema:
+            type: string
+          example: iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx
+          description: Bech32-encoded address that should be searched for.
```

Comment:
Maybe better to clarify, that it's only returning outputs with this address in an AddressUnlockCondition